share/picongpu/examples/LaserWakefield/etc/picongpu/4.cfg

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2018 Axel Huebl, Rene Widera, Felix Schmitt
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="1:00:00"
+
+TBG_devices_x=1
+TBG_devices_y=4
+TBG_devices_z=1
+
+TBG_gridSize="128 2048 128"
+TBG_steps="2048"
```

Comment:
please add sliding window

Reply:
since that's a similar case to the `1.cfg`, just multi-GPU, I would leave this without moving window.